### PR TITLE
[pt-br] Chapter01

### PR DESCRIPTION
**Pull request by kmacedovarela**
#  [ptBR]  Chapter01

- :gift: Adding portuguese translation

In addition, categorize the changes you're making using the "/kind" Prow command, example:

/kind documentation

**Release Note**

```release-note 

- Chapter 1 pt-br translation

```

Contributes to #26

### CODE REVIEW

**Comment by salaboy** [comment]
@kmacedovarela, this is awesome.. should we rename the file to pt_BR.md? so we follow the same approach with all languages.. I know that Matheus @mcruzdev wanted also to help.. so it might be good to distribute a bit the work.

**Comment by kmacedovarela** [comment]
We should absolutely follow a pattern of your choice. Unfortunately, I could not identify any ootb i18n feature for readme markdown files in GitHub, so it should be chosen based on trade-offs and your preferences   :) 

Things to consider....
-  the book's target audience of readers [k8s]
-  contributors[k8s, cloud-native], 
-  i18n industry standards, 
-  [k8s docs](https://github.com/kubernetes/website/tree/main) chosen approach,
- clarity of assets names,
-  and the fact that currently there is no more than one translation per language ( many country areas, e.g. pt_BR, pt_PT, zn_CN, zn_TW, and so on...)

...I'd suggest either

1. README.md, README-pt.md , README-zn.md. _[my favorite, after some extra research]_
2. README.md, README_pt.md , README_zn.md

Eventually if you get new translation prs from conflicting country areas (or if you already expect it), then you could reevaluate the strategy. =)

The only option I do 👎  is using the current option with two dots [README.pt-br.md], so thanks for considering other options :D

**Comment by salaboy** [comment]
I agree let's use the kubernetes approach with two letters from the iso like in this repo https://github.com/kubernetes/website/tree/main

File: chapter-1/README.pt-br.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Capítulo 1 :: (A ascensão das) Plataformas Baseadas em Kubernetes
+
+Graças à fantástica comunidade cloud-native, você tem acesso a estes tutoriais nas seguintes línguas:
+- [Chinese `zh-cn`](README.zh-cn.md) :cn:
+- [Portuguese (Brazil) `pt-BR`](README.zh-cn.md) :brazil:
```
Comment (salaboy) [inline chapter-1/README.pt-br.md]: @kmacedovarela pointing to the wrong file here

**Comment by mcruzdev** [comment]
Hi @kmacedovarela @salaboy I am off today and tomorrow, but on Monday I want to help surely.

**Review by salaboy — APPROVED** [review]

**Pull request merged**